ReactWindows/ReactNative/UIManager/OnLayoutEvent.cs

```diff
@@ -6,12 +6,12 @@ namespace ReactNative.UIManager
 {
     class OnLayoutEvent : Event
     {
-        private int _x;
-        private int _y;
-        private int _width;
-        private int _height;
+        private double _x;
```

Comment:
@rozele makes sense to use the Dimensions class here as it is extensible and not just a bunch of params